README.md

```diff
@@ -1,62 +1,86 @@
+[![](https://user-images.githubusercontent.com/1248214/57789522-600fcc00-7739-11e9-86d9-73d7032f40fc.png)
+](https://kilt.io)
 
-![](https://user-images.githubusercontent.com/1248214/57789522-600fcc00-7739-11e9-86d9-73d7032f40fc.png)
+Data sovereignty and interoperability
 
-# KILT Testnet (Mash-net) SDK
+# Mash-Net SDK
 
-The KILT SDK is a collection of classes and methods that application developers can utilize to interact with the KILT Network. The SDK is provided in Typescript.
+The open-source KILT's SDK testnet is written in TypeScript and enables developers to build, play and design permissionless blockchain apps and businesses. [KILT](https://kilt.io) network provides a system for self-sovereign data and interoperability. The SDK provides collection of classes and methods developers can utilize to interact with the KILT Network.
 
-Read the [getting started guide](./docs/getting-started.md), or browse the [API documentation](https://kiltprotocol.github.io/sdk-js/api).
+- **Self-sovereign data.** Have ownership of your digital and analog identities, with control over who you share that data with. Providing that extra layer of flexiblity and security.
```

Comment:
Wording suggestion: use "your users" instead of "you", since the README is for devs

Reply:
Yes, you are right. We are doing the Readme for devs intended for the end user